docs/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "docs",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "files": [
+        "dist",
+        "tailwind.config.js"
+    ],
```

Comment:
```suggestion
```

Reply:
Not solved, we don't need to expose

Reply:
what exactly is the change here? github shows the exact sams snippe twice

Reply:
It is deleting the `files` array as we don't need to expose those files (it is not a npm package to publish)